libs/blocks/merch-card/merch-card.js

```diff
@@ -0,0 +1,205 @@
+      const standardWrapper = createTag('div', { class: 'standard-wrapper' });
+      const secureTransactionWrapper = decorateWithSecureTransactionSign(secureTransactionSign);
+      standardWrapper.append(secureTransactionWrapper, cardFooterRow);
+      cardFooter?.append(standardWrapper);
+      secureTransactionSign.remove();
+    }
+
+    const altCta = Array.from(el.querySelectorAll('div > div[data-align="center"][data-valign="middle"]')).filter((data) => data.textContent?.trim() === 'alt-cta');
+    const altCtaMetaData = altCta[0]?.parentElement?.nextElementSibling?.querySelectorAll('div > div');
+    const altCtaRegex = /href="([^"]*)"/g;
+    if (altCtaMetaData?.length === 2 && altCtaMetaData[1]?.innerHTML?.match(altCtaRegex)) {
+      const originalCtaButton = cardFooterRow.querySelector('.consonant-CardFooter-cell--right');
+      const altCtaButton = originalCtaButton.cloneNode(true);
+      const checkboxContainer = createCheckbox(altCtaMetaData[0]);
+
+      altCtaButton.innerHTML = altCtaMetaData[1].innerHTML;
+      altCtaButton.classList.add('button--inactive');
+      checkboxContainer.querySelector('input[type="checkbox"]').addEventListener('change', (event) => {
+        if (event.target.checked) {
+          toggleButtonActiveState(originalCtaButton, altCtaButton);
+        } else {
+          toggleButtonActiveState(altCtaButton, originalCtaButton);
+        }
+      });
+      cardFooterRow.append(altCtaButton);
+      cardFooter.prepend(checkboxContainer);
+      altCta[0].parentNode.remove();
+      altCtaMetaData[0].parentNode.remove();
+    }
+  };
+
+  decorateAlternativeCta();
+  cardFooter.querySelectorAll('.consonant-CardFooter-cell').forEach((cell) => cell.classList.add(`consonant-${cardType}-cell`));
+};
+
+const addInner = (el, cardType, merchCard) => {
+  const titles = Array.from(el.querySelectorAll('h1, h2, h3, h4, h5, h6'));
+  const rows = Array.from(el.querySelectorAll('p'));
+  const styles = Array.from(el.classList);
+  const merch = styles.includes('merch-card');
+  const pElement = merch && el.querySelector(':scope > div > div > p:last-of-type');
+  const links = pElement ? pElement.querySelectorAll('a') : el.querySelectorAll('a');
+
+  const inner = el.querySelector(':scope > div:not([class])');
+  inner.classList.add(`consonant-${cardType}-inner`);
+  const title = createTitle(titles, cardType);
+  const description = createDescription(rows, cardType, inner);
+
+  inner.prepend(title);
+  inner.append(description);
+  addFooter(links, inner, merchCard);
+  decorateFooter(el, cardType);
+  merchCard.append(inner);
+};
+
+const decorateRibbon = (el, cardType) => {
+  const ribbonMetadata = el.querySelectorAll('div > div[data-align="center"][data-valign="middle"]');
+
+  if (ribbonMetadata.length >= 2) {
+    const ribbonStyle = ribbonMetadata[0].outerText;
+    const ribbonStyleRegex = /^#[0-9a-fA-F]+, #[0-9a-fA-F]+$/;
+    if (!ribbonStyleRegex.test(ribbonStyle)) return;
```

Comment:
Is there a default ribbon style, if no color values are passed and a label is? What about if just one color value is passed?
<img width="658" alt="Screen Shot 2023-08-22 at 4 41 03 PM" src="https://github.com/adobecom/milo/assets/2086146/8aa30291-0259-41ea-9c70-cc9ddbba14bd">

Reply:
<img width="1241" alt="Screen Shot 2023-08-22 at 5 10 04 PM" src="https://github.com/adobecom/milo/assets/2086146/9bda9b95-47a0-4676-a02e-b48326cb026e">

testing authoring regression fyi

Reply:
Was this actually fixed or did the comment just get 'resolved'? 
Please communicate your intentions and resolves.